### ISSUE TITLE
Loss Function

### ISSUE THREAD

**Issue by deep0learning** [mingyuliutw/UNIT#43]
In which file you have defined loss function such as  mean squared error (MSE) and L2 for domain adaptation ?

**Comment by mingyuliutw** [comment]
Please check pytorch documentation and you will find out the implementation.

**Comment by mingyuliutw** [comment]
Please check pytorch documentation and you will find out the implementation.
